src/pattern-library/components/patterns/UsingComponentsPage.js

```diff
@@ -90,152 +85,207 @@ export default function UsingComponentsPage() {
 
           <p>
             Presentational components also take <strong>HTML attributes</strong>{' '}
-            applicable to their outermost element.
+            applicable to their primary element, typically the outermost element
+            rendered by the component.
           </p>
-        </Library.Example>
+        </Library.Pattern>
 
-        <Library.Example title="Base presentational components">
+        <Library.Pattern title="Base presentational components">
           <p>
-            <strong>Base presentational components</strong> extend
-            presentational component props with an optional boolean{' '}
-            <code>unstyled</code> prop. When set, the component will have no
-            styling at all, allowing for complete customization.
+            <strong>Base presentational components</strong> are similar to
+            presentational components, but allow more extensive customization.
           </p>
-        </Library.Example>
-
-        <Library.Example title="Composite components">
           <p>
-            <strong>Composite components</strong> accept the same common props
-            as presentational components with the exception of{' '}
-            <code>classes</code>.
+            Example: <code>ButtonBase</code>, <code>LinkBase</code>.
           </p>
-        </Library.Example>
 
-        <Library.Example title="Simple components">
           <p>
-            <strong>Simple components</strong> take only documented,
-            component-specific props.
+            By default, base presentational components apply some core styling.{' '}
+            <strong>Base styling might include</strong> things like basic layout
+            (e.g. establishing a <code>flex</code> layout), focus rings,
+            establishing transition <em>properties</em> or preventing text from
```

Comment:
Not sure what the italics are for here.

Reply:
To emphasize that base styling might establish transition "settings" (e.g. which properties transition) but not the property values themselves (e.g. base styles might set up all colors to transition, but won't define what those colors are). Any ideas how better to word this?